generate_buddy_headers.py

```diff
@@ -22,6 +22,7 @@
 """#pragma once
 #include "inttypes.h"
 #include "button_operations.h"
+#include "error_types.h"
```

Comment:
Nitpick: I'd name the file `"error_type.h"`

Reply:
force-pushed